packages/@azure/servicebus/data-plane/lib/session/messageSession.ts

```diff
@@ -493,16 +479,19 @@ export class MessageSession extends LinkEntity {
   }
 
   /**
-   * Receives messages from a session enabled entity.
-   * @param onMessage The message handler to receive service bus messages from a session
-   * enabled entity.
-   * @param onError The error handler to receive an error that occurs while receiving messages
-   * from a session enabled entity.
+   * Starts the receiver in streaming mode by establishing an AMQP session and an AMQP receiver
+   * link on the session.
+   * To stop receiving messages, call `close()` on the SessionClient.
+   *
+   * @param onSessionMessage Callback for each incoming message.
+   * @param onError Callback for any error that occurs while receiving messages.
+   * @param options - Options to control whether messages should be automatically completed and/or
+   * automatically have the session lock renewed.
```

Comment:
Why doesn't this not return a receive handler? 
Were we thinking of moving to passing in timeout as a parameter-ish thing?
Either way, I would prefer this to mirror/be similar to original streaming receiver's signature.